.ci/run.sh

```diff
@@ -79,6 +79,10 @@ case "${CI_JOB}" in
 		info "Running Confidential Containers tests for AMD SEV"
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make cc-sev-kubernetes"
 		;;
+	"CC_SNP_CRI_CONTAINERD_K8S")
```

Comment:
Sorry, just notice this `CC_SNP_CRI_CONTAINERD_K8S`.... usually we have `CC_CRI_CONTAINERD_K8S_SOME_SPECIFIC_CONFIG`. So it should be `CC_CRI_CONTAINERD_K8S_SNP`.

`CC_SEV_CRI_CONTAINERD_K8S` above seems wrong but better not change it right now.

Reply:
@wainersm changing this will need changes to be made in the CI repo as well. So do you think it is safer to do that in a different PR? I can open one as soon as this gets merged. I already have a PR open in ci repo. What do you think?

Reply:
@UnmeshDeodhar Indeed it is safer to do that change on a follow up PR.